src/main/java/com/google/devtools/build/lib/remote/RemoteCache.java

```diff
@@ -102,9 +103,15 @@ public RemoteCache(
   public ActionResult downloadActionResult(
```

Comment:
Please change the return value to `CachedActionResult` instead of adding another download method.

Reply:
done